src/index.html

```diff
@@ -7,8 +7,96 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__catalog logo">
```

Comment:
Add `href` attribute to the anchor tag containing the logo to make the logo clickable and link it to the homepage. Example: `<a class="header__catalog logo" href="/">`